test/convert.cc

```diff
@@ -343,10 +343,6 @@ int main(int argc, char** argv)
   mesh->verify();
   mesh->writeNative(smb_path);
 
-  auto ft = mesh->findTag("fathers2D");
```

Comment:
@KennethEJansen I assume removing this is OK?  The other `convert` tests don't have the "fathers2d" tag.